packages/create-redwood-app/template/api/src/functions/graphql.ts

```diff
@@ -6,9 +6,11 @@ import {
 
 import schemas from 'src/graphql/**/*.{js,ts}'
 import { db } from 'src/lib/db'
+import { logger } from 'src/lib/logger'
 import services from 'src/services/**/*.{js,ts}'
 
 export const handler = createGraphQLHandler({
+  logger,
```

Comment:
With this PR, GraphQL logging will be possible (and via either Apollo or Envelop), correct?

If so, note that we'll want to update the Logging doc.